src/main/java/com/academy/task/util/ConnectionProperties.java

```diff
@@ -0,0 +1,20 @@
+package com.academy.task.util;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@Getter
+@Setter
+@ConfigurationProperties(prefix = "db")
+public class ConnectionProperties {
+
+    String driver;
+
+    String URL;
+
+    String USER;
+
+    String PASS;
```

Comment:
Java naming convention states that non constant field of class should be named in camel case.